payment/key.go

```diff
@@ -0,0 +1,104 @@
+package payment
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"encoding/hex"
+	"fmt"
+	"math/rand"
```

Comment:
Please do not use math/rand for generating preshared-keys random data.  Use crypto/rand.

Reply:
Whoops! My editor auto-formats and imported this library instead of the correct one 
https://github.com/brave-intl/bat-go/pull/285/commits/f5ca28286a4358dce3ee80d23a3f1c8bdc66e717